ccm_web/client/src/pages/AddNonUMUsers.tsx

```diff
@@ -117,38 +117,35 @@ export default function AddNonUMUsers (props: AddNonUMUsersProps): JSX.Element {
     )
   }
 
-  const resetFeature = (): void => setActivePageState(PageState.SelectInputMethod)
+  const resetFeature = (): void => {
```

Comment:
As mentioned [here](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/288/files#r786158224) Probably this is true here as well since the BE is not integrated, I cannot say it.